city_scrapers/spiders/wicks_bocc.py

```diff
@@ -0,0 +1,112 @@
+from urllib.parse import urljoin
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class WicksBoccSpider(CityScrapersSpider):
+    name = "wicks_bocc"
+    agency = "Board of Sedgwick County Commissioners"
+    timezone = "America/Chicago"
+    # original https://www.sedgwickcounty.org/commissioners/commission-meetings/
+    # original page embeds iframe with the following URL
+    # we scrape the embedded URL instead
+    start_urls = ["https://sedgwick.granicus.com/ViewPublisher.php?view_id=34"]
+    location = {
+        "name": "Ruffin Auditorium",
+        "address": "100 N. Broadway, Lower Level, Wichita, KS 67202",
+    }
+
```

Comment:
You are right. I assumed that if the tests were passing the crawl command would also work. 
```
scrapy crawl wicks_bocc -O test_output.csv
``` 
attempts to scrape but is prevented by `robots.txt` file. However, the generator used to create the spider did not have any problems: 
```
scrapy genspider wicks_bocc "Board of Sedgwick County Commissioners" https://sedgwick.granicus.com/ViewPublisher.php?view_id=34
```
Should the generator be checking the `robots.txt` file as well? If not, then maybe updating the documentation about how to go around robots file could help.

Reply:
No worries, @cruznunez! And yeah, I hadn't considered whether [genspider](https://github.com/City-Bureau/city-scrapers-core/blob/main/city_scrapers_core/commands/genspider.py) should catch the `robots.txt` issue. It would be nice if the command automatically added the robots.txt ignore setting to the spider if it detects that it'll be a problem. I'm going to add it to my "to do" list. Thanks for the suggestion 👍